rest/results-tabulation-api/deployment-guide.md

```diff
@@ -0,0 +1,73 @@
+# Deploying tabulation BACKEND on production environments
+
+## Prerequisites
+
+Production environment created by following https://github.com/ECLK/Deployment
+WSO2 API Manager with IS as km
+Platformer cloud account connected to Prod k8s cluster for CI/CD
+Deploy https://github.com/ECLK/auth-app using platformer cloud
+
+##  Application Setup
+
+### On Platformer cloud
+
+* Create an app for tabulation backend
+* Connect ECLK/results-tabulation-api in the Repository section
+* Create appropriate BUILD CONFIG
+    * Dockerfile path - ./rest/results-tabulation-api/Dockerfile
+    * Build context - ./rest/results-tabulation-api
+* Setup Environment variables on RUN TIME
+    * ENV_CONFIG - path to the environment config file
+* Add Config file on CONFIG FILES
+```
+DEBUG = False
+
+DATABASE_PLUGIN = "mysql+pymysql"
+DATABASE_USERNAME = ""
+DATABASE_PASSWORD = ""
+DATABASE_HOST = ""
+DATABASE_PORT = ""
+DATABASE_NAME = ""
+
+
+RESULT_DISSEMINATION_SYSTEM_URL = ""
+RESULT_DISSEMINATION_SYSTEM_ELECTION_CODE = "2019PRE"
+RESULT_DISSEMINATION_SYSTEM_RESULT_TYPE_VOTE = "PRESIDENTIAL-FIRST"
+RESULT_DISSEMINATION_SYSTEM_RESULT_TYPE_PREF = "PRESIDENTIAL-PREF"
+
+PROD_ENV = True
+
+```
+* Mount a volume to be persistent
+    * PATH - /app/data
```

Comment:
This is no longer used since all of them has been moved to db. So, this is not needed.

Reply:
Removed